build/v1/types.go

```diff
@@ -86,6 +86,13 @@ type CommonSpec struct {
 	// are ignored.
 	// +optional
 	NodeSelector OptionalNodeSelector `json:"nodeSelector" protobuf:"bytes,9,name=nodeSelector"`
+
+	// mountProxyTrustedCA bind mounts the cluster's trusted certificate authorities,
+	// as defined in the cluster's proxy configuration, into the build. When this field is set to
+	// true, the contents of `/etc/pki/ca-trust` within the build are managed by the build
+	// container and cannot be modified during the build (example - via a Dockerfile `RUN`
+	// instruction).
```

Comment:
they can be modified, but the files (neither the original nor the modified versions) will be included in the output image.

the output image will contain whatever contents the original from image included for /etc/pki/ca-trust